packages/components/src/integration/iot-resource-explorer/iot-resource-explorer.spec.component.ts

```diff
@@ -28,13 +25,13 @@ it('sort by name asc', () => {
   cy.matchImageSnapshotOnCI('sort by name asc');
 });
 
-it('filter by name', () => {
-  renderComponent();
-  cy.get(`.${testContainerClassName}`).should('be.visible');
-  setTextFilterValue('Turbine', '[placeholder="Filter by name"]');
-  cy.waitUntil(() => cy.get(getTableRowsSelector()).then((rows) => rows.length === 1));
-  cy.matchImageSnapshotOnCI('filter by name');
-});
+// it('filter by name', () => {
```

Comment:
Why are these commented?

Reply:
Ahh those should not be, that was just for testing purposes. I will clean that up